pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -114,8 +114,8 @@ func (podDisruptionBudgetStatusStrategy) ValidateUpdate(ctx context.Context, obj
 	var apiVersion schema.GroupVersion
 	if requestInfo, found := genericapirequest.RequestInfoFrom(ctx); found {
 		apiVersion = schema.GroupVersion{
-			Group:   requestInfo.APIVersion,
-			Version: requestInfo.APIGroup,
+			Group:   requestInfo.APIGroup,
+			Version: requestInfo.APIVersion,
```

Comment:
this is exciting... can you add a unit test that would have caught this?

Reply:
Yeah, I found this because of the unit test.